stix2/test/test_custom.py

```diff
@@ -94,6 +94,26 @@ def test_custom_property_in_bundled_object():
     assert '"x_foo": "bar"' in str(bundle)
 
 
+def test_custom_marking_no_init():
+    @stix2.CustomMarking('x-new-obj', [
+        ('property1', stix2.properties.StringProperty(required=True)),
+    ])
+    class NewObj():
+        pass
+
+    no = NewObj(property1='something')
+    assert no.property1 == 'something'
+
+    @stix2.CustomMarking('x-new-obj2', [
```

Comment:
let's split this into two tests.

Reply:
Then we should also split the analogous tests for CustomObject, CustomObservable, and CustomExtension.